torchmetrics/collections.py

```diff
@@ -207,7 +207,7 @@ def _to_renamed_ordered_dict(self) -> OrderedDict:
             od[self._set_name(k)] = v
         return od
 
-    def keys(self, keep_base: bool = False):
+    def keys(self, keep_base: bool = False) -> Iterable[str]:
```

Comment:
```suggestion
    def keys(self, keep_base: bool = False) -> Iterable[Hashable]:
```

keys aren't necessarily strings